src/components/waves/waves.ts

```diff
@@ -0,0 +1,137 @@
+let i = 0;
+const uid = () => (++i).toString();
```

Comment:
Use id generation like in tab component `Math.random().toString().split('.')[1];`